src/Writers/HTMLWriter.jl

```diff
@@ -1030,7 +1030,11 @@ mdconvert(h::Markdown.Header{N}, parent; kwargs...) where {N} = DOM.Tag(Symbol("
 
 mdconvert(::Markdown.HorizontalRule, parent; kwargs...) = Tag(:hr)()
 
-mdconvert(i::Markdown.Image, parent; kwargs...) = Tag(:img)[:src => i.url, :alt => i.alt]
+mdconvert(i::Markdown.Image, parent; kwargs...) = if endswith(i.url, ".webm")
+    Tag(:video)[:src => i.url, :width => "800", :controls => "true", Symbol("data-title") => i.alt]
```

Comment:
I am wondering about `data-title` -- why not just use `title`? The latter is a [global attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title), so it should be correct to apply to a `<video>` tag too?